analyzer/src/funTest/assets/projects/synthetic/gradle-all-dependencies-expected-result-with-curations.yml

```diff
@@ -48,6 +48,8 @@ analyzer:
     tool_versions: {}
   config:
     allow_dynamic_versions: false
+    enabled_package_managers:
```

Comment:
commit-message: I don't understand which cache is referred to by `caching`. Could you mention that explicitly?

Reply:
I've simply moved the word "Gradle" more to the front which hopefully makes in clearer that this is all about Gradle.